infrastructure/local-setup-preparation/src/index.ts

```diff
@@ -31,34 +34,51 @@ async function depositWithRichAccounts() {
     );
 
     for (const wallet of wallets) {
-        const contract = new ethers.Contract(process.env.CONTRACTS_BRIDGEHUB_PROXY_ADDR, utils.BRIDGEHUB_ABI, wallet);
+        if (!(wallet.address === OPERATOR)) {
```

Comment:
If there a 'getOperateKey' is used, it should also  be used for the actual operator, this means: adding it to ValidatorTimelock in deploy.ts and adding it to the env vars, 
`ETH_SENDER_SENDER_OPERATOR_COMMIT_ETH_ADDR`
`ETH_SENDER_SENDER_OPERATOR_BLOBS_ETH_ADDR`

This might be a bit different for local-setup though.